main.go

```diff
@@ -28,11 +29,23 @@ func abort(writer http.ResponseWriter) error {
 }
 
 func startLogServe(stream *stream.Stream) {
+	// Get access token from environment variable
+	accessToken := os.Getenv("ACCESS_TOKEN")
+
 	routes := http.NewServeMux()
-	routes.HandleFunc("/log", func(w http.ResponseWriter, r *http.Request) {
+	routes.HandleFunc("/log/", func(w http.ResponseWriter, r *http.Request) {
 		debug("output %s %s", r.Method, r.URL.String())
-		// TODO: Add method context switching here...
-		getLog(stream, w, r)
+
+		// Authenticate the request with accessToken, this is good enough because
+		// live logs are short-lived
+		if (r.URL.String()[5:] != accessToken) {
```

Comment:
maybe just a comment above this line to explain the r.URL.String() is "/log/" + accessToken, so skip first 5 chars?